website/docs/migration/premigration-testing.md

```diff
@@ -22,8 +22,25 @@ As an example, let us say that your application performs some complex query and
 You would do the following:
 
 1. Start FerretDB in `diff-normal` mode.
-   This can be achieved by using the `--mode` [flag](../configuration/flags.md) or by setting the `FERRETDB_MODE` environment variable.
-2. Run `mongosh` and insert some documents.
+
```

Comment:
What part of the instruction is automated testing? Just seeing the header, I expected there were some references to automated testing.

Reply:
It's implied that manual, is testing an application by running queries from the shell or running some component of an application. Automated implies running whatever tests the user has written themselves etc. I suppose we could omit this and have the header be "Testing with `diff-normal` mode".